### PR TITLE
Improved sentence

### PR DESCRIPTION
**Pull request by edsonmedina**

### CODE REVIEW

**Comment by timglabisch** [comment]
:+1:

File: best_practices/configuration.rst [hunk 1]
```diff
@@ -101,7 +101,7 @@ to control the number of posts to display on the blog homepage:
     parameters:
         homepage.num_items: 10
 
-If you ask yourself when the last time was that you changed the value of
+If you ask yourself when was the last time you changed the value of
```
Comment (weaverryan) [inline best_practices/configuration.rst]: Ah, I don't love the new one either. The problem is the sentence is *almost* a question, but not *really* a question, so it's funny both ways. So how about something totally different:

```rst
If you've done something like this in the past, it's likely that you've in fact *never* actually needed
to change that value.
```

What do you think @edsonmedina?

Comment (WouterJ) [inline best_practices/configuration.rst]: :+1:

**Comment by xabbuh** [comment]
@edsonmedina Could you please make the suggested changes? :)

**Comment by edsonmedina** [comment]
Done

**Comment by xabbuh** [comment]
@edsonmedina If you now could please change the line wrapping to wrap them after the first word that crosses the 72nd character, your pull request will be perfect. :)

**Comment by edsonmedina** [comment]
@xabbuh check it now

**Comment by WouterJ** [comment]
It's perfect now. Thanks @edsonmedina!

**Comment by xabbuh** [comment]
:+1:

**Comment by weaverryan** [comment]
Thanks Edson for the PR and working with us with the updates. It's very nice now!

**Pull request closed without merging**